configure.ac

```diff
@@ -170,6 +170,7 @@ AC_CONFIG_FILES([Makefile
                  pyanaconda/modules/security/Makefile
                  pyanaconda/modules/timezone/Makefile
                  pyanaconda/modules/network/Makefile
+                 pyanaconda/modules/firewall/Makefile
```

Comment:
Here should be `pyanaconda/modules/network/firewall/Makefile`.

Reply:
Fixed.